src/actions/redeemPointsAction.js

```diff
@@ -84,11 +84,13 @@ export const fetchRedemptionsSuccess = redemptions => ({
  * @param {String} societyId
  * @returns {(dispatch) => Promise<AxiosResponse>}
  */
-export const fetchRedemption = societyId => (
-  (dispatch) => {
+export const fetchRedemption = (ref) => {
```

Comment:
Nitpick: No need for the brackets around `ref`

Reply:
airbnb elsint rules: [eslint] Expected parentheses around arrow function argument having a body with curly braces. (arrow-parens)

Reply:
Noted